Create deg2rad.md (#5726)
Codecademy/docs: content/numpy/concepts/math-methods/terms/deg2rad/deg2rad.md

---
Title: '.deg2rad()'
Description: 'Converts angles from degrees to radians.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
  - 'Web Development'
Tags:
  - 'Math'
  - 'NumPy'
CatalogContent:
- 'learn-python-3'
- 'paths/computer-science'
---

> In NumPy, the **`.deg2rad()`** function converts angles from degrees to radians.

Reviewer: To make it clearer, we should rephrase the first sentence to:

```suggestion
In NumPy, the **`.deg2rad()`** function converts an angle from degrees to radians.
```

> > **Note:** In NumPy, the default unit for angles is radians. Therefore, the `.deg2rad()` function converts degree inputs to radians when applied.

Reviewer: Second sentence can be simplified as follows:

```suggestion
> **Note:** In NumPy, the default unit for angles is radians. Therefore, the `.deg2rad()` function is used to convert angle values from degrees to radians.
```
